src/libraries/System.Net.NetworkInformation/Directory.Build.props

```diff
@@ -3,6 +3,6 @@
   <PropertyGroup>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
-    <UnsupportedOSPlatforms>browser</UnsupportedOSPlatforms>
+    <UnsupportedOSPlatforms>browser;android</UnsupportedOSPlatforms>
```

Comment:
Why is it marking the whole assembly unsupported when only a few methods seem to be not supported?

Reply:
Hm, perhaps I misread my own comment and used the first line as a separate point

Reply:
I guess that's why 
```
<GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsAnyOS)' == 'true'">SR.SystemNetNetworkInformation_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
```

Reply:
If this namespace is expected to work on Android in general then I'll mark some particular methods only.

Reply:
Yes, a lot of the library is expected to work on Android.